docs/.markdownlint-cli2.cjs

```diff
@@ -99,6 +99,7 @@ module.exports = {
             "style": "atx",
         },
         "line-length": false, // MD013
+        "link-image-reference-definitions": false, // MD053
```

Comment:
Is there a specific reason we have to disable this? If so, can we document it here?